extensions/dict_SWm/dict_SWm.did

```diff
@@ -0,0 +1,38 @@
+dict_rfc6942_avps
```

Comment:
Is this standard syntax of free diameter of writing supported dictionary ?

Reply:
Yes.  This code was generated using an internal tool that takes the definitions from the specs and generates the dictionary code.  Over 1/2 of the dictionaries that are in the extensions directory were generated with this tool.